build/js/live-editor.output_webpage.js

```diff
@@ -683,6 +683,27 @@ window.WebpageOutput = Backbone.View.extend({
             return deferred;
         }
 
+        if (results.warnings && results.warnings.length > 0) {
+            var warnings = [];
+            for (var i = 0; i < results.warnings.length; i++) {
+                var pos = results.warnings[i].parseInfo.cursor || 0;
+                var previous = userCode.slice(0, pos);
+                var column = pos - previous.lastIndexOf("\n") - 1;
+                var row = (previous.match(/\n/g) || []).length;
+
+                warnings.push({
+                    row: row,
+                    column: column,
+                    text: this.getLintMessage(results.warnings[i].parseInfo),
+                    type: "warning",
+                    source: "slowparse"
+                });
+            }
+
+            deferred.resolve(warnings);
+            return deferred;
```

Comment:
It may be possible to have warnings and an error if the warnings comes before the error.  The code should handle this case by returning the warnings and the error in the same array.